src/style.css

```diff
@@ -1 +1,50 @@
 /* write your styles here */
+body {
+  margin-top: 0;
+}
+
+#header {
+  background-color: blue;
+  width: 300px;
+  height: 100px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.important {
+  background-color: red;
+  width: 300px;
+  height: 100px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.interesting {
+  background-color: green;
+  width: 300px;
+  height: 100px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.surprising {
+  background-color: orange;
+  width: 300px;
+  height: 100px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#footer {
+  background-color: black;
+  width: 300px;
+  height: 100px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.important:hover,
+.interesting:hover,
+.surprising:hover {
+  height: 200px;
+}
```

Comment:
.block1, .block2, .block 3 {
   // write common styles there
}

.block1 {
  // unique styles for block 1
}

.block1 {
  // unique styles for block 2
}

Don't repeat yourself, you can write it shorter